### PR TITLE
Persist elevation plot settings

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: model/shared/src/main/scala/explore/model/ElevationPlotOptions.scala [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+object ElevationPlotOptions {
+  val site  = Focus[ElevationPlotOptions](_.site)
+  val range = Focus[ElevationPlotOptions](_.range)
+  val time  = Focus[ElevationPlotOptions](_.time)
+
+  val Default =
+    ElevationPlotOptions(Site.GS, PlotRange.Night, TimeDisplay.Site)
+
+  implicit val targetVisualOptionsEq: Eq[ElevationPlotOptions] =
```
Comment (toddburnside) [inline model/shared/src/main/scala/explore/model/ElevationPlotOptions.scala:26]: Someday names won't be necessary. But for now this should probably match the class. 😄

**Review by toddburnside — APPROVED** [review]
LGTM

File: model/shared/src/main/scala/explore/model/ElevationPlotOptions.scala (hunk 1 above)
Comment (cquiroz) [reply model/shared/src/main/scala/explore/model/ElevationPlotOptions.scala:26]: Some day as in scala 3 auto derivation?

Comment (toddburnside) [reply model/shared/src/main/scala/explore/model/ElevationPlotOptions.scala:26]: Well, that too. But I don't think `given`s have to have names in scala 3

**Pull request merged**